packages/fxa-auth-server/scripts/stripe-products-and-plans-to-firestore-documents/stripe-products-and-plans-converter.ts

```diff
@@ -348,6 +374,55 @@ export class StripeProductsAndPlansConverter {
     return planConfig;
   }
 
+  /**
+   * Validate the ProductConfig object, and if its valid write the object to JSON.
+   */
+  async writeToFileProductConfig(
+    productConfig: ProductConfig,
+    existingProductConfigId: string | null,
+    path: string
+  ) {
+    await this.paymentConfigManager.validateProductConfig(productConfig);
+    await fsPromises.writeFile(
+      path,
+      JSON.stringify({
+        ...productConfig,
+        id: existingProductConfigId,
+      })
+    );
+    this.log.debug(
+      'StripeProductsAndPlansConverter.writeToFileProductSuccess',
+      {
+        path,
+      }
+    );
+  }
+
+  /**
+   * Validate the PlanConfig object, and if its valid write the object to JSON.
+   */
+  async writeToFilePlanConfig(
+    planConfig: PlanConfig,
+    productConfigId: string,
+    existingPlanConfigId: string | null,
```

Comment:
Sorry I mis-read this on the initial review.

However, it seems like these ids are either existing (exporting Firestore) or null (exporting Stripe metadata as config docs).  Since this script is exporting metadata to config jsons, it should always be null?